test/e2e/apimachinery/flowcontrol.go

```diff
@@ -248,33 +267,35 @@ func createPriorityLevel(f *framework.Framework, priorityLevelName string, assur
 	}
 }
 
-//lint:ignore U1000 function is actually referenced
-func getPriorityLevelConcurrency(f *framework.Framework, priorityLevelName string) int32 {
+func getPriorityLevelConcurrency(f *framework.Framework, priorityLevelName string) (int32, error) {
```

Comment:
```suggestion
func getPriorityLevelConcurrency(c kubernetes.Interface, priorityLevelName string) (int32, error) {
```
we dont need the whole framework reference if we're returning an error in this method, amiright?